CHANGELOG.adoc

```diff
@@ -17,13 +17,21 @@ Given a version number MAJOR.MINOR.PATCH:
 == v3.0.0 (Unreleased)
 
 === Breaking changes
-* Set minimum Terraform version to 1.0.0 (fixes #49)
+* Changed minimum Terraform version to 1.0.0 (fixes #49)
+* Deprecated `vcn_cidr`, use `vcn_cidrs` instead (list of IPv4 CIDRs).
+* [ ] Deprecated previous gateway creation variable names. We now use imperative style, see codingconventions (fixes #24)
```

Comment:
Same as for the vcn_cidrs - as this will be a major release, let's take the opportunity to rename the control variables, especially the gateway control variables accordingly. I'm hoping we can make the external api more stable after this.

We can do a number of pre-releases to give users the opportunity to test and upgrade to 3.x.

Reply:
This is the goal in the next change. What you see here is just the changelog I prepared with the todo list for [3.0.0 milestone](https://github.com/oracle-terraform-modules/terraform-oci-vcn/milestone/3).

This PR fix #21 and #51
The next one will fix #24 #30 and #44 

At that point, 3.0.0 should be ready for release.